Formula/inadyn.rb

```diff
@@ -0,0 +1,41 @@
+class Inadyn < Formula
+  desc "Dynamic DNS client with IPv4, IPv6, and SSL/TLS support"
+  homepage "http://troglobit.com/projects/inadyn/"
+  url "https://github.com/troglobit/inadyn/releases/download/v2.5/inadyn-2.5.tar.gz"
+  sha256 "f7faf0be4f0b4bfa1acc811189a9ed0a58bc367e48ea31c283920a2ef27cdc40"
+
+  patch do
+    url "https://github.com/troglobit/inadyn/commit/57bdcc0321b49ee68397c70140d9895655edb06f.patch"
+    sha256 "ef9a5bda8ff3ef9c221a15859adeeac4c5baff2c2cfe34ee90d91595091bc3c3"
+  end
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "cmake"    => :build
+  depends_on "libtool"  => :build
+  depends_on "confuse"
+  depends_on "gnutls"
+  depends_on "pkg-config"
+
+  def install
+    mkdir_p buildpath/"inadyn/m4"
+    system "autoreconf", "-W", "portability", "-vif"
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}",
+                          "--sysconfdir=#{etc}",
+                          "--localstatedir=#{var}"
+    system "make", "install"
+  end
+
+  def caveats; <<-EOF
+    Place your DDNS provider's parameters in #{etc}/inadyn.conf
```

Comment:
The caveat is generic advice about using the software, and not Homebrew specific I guess. If so, please remove it.

Reply:
The paths are homebrew specific, in particular where to find the template files. I'm happy to rephrase or approach this however you want, but expecting the user to grep around the filesystem or use `brew list inadyn` to find the config files seems very unfriendly. A great many formula provide instructions on how to get started.

Again, happy to provide in whatever manner you feel is best.

Reply:
They are standard paths that are commonly used within Homebrew. The caveat is unnecessary, please remove it.